File: iOS Examples/Objective-C/EffectsProcessorDemo/EffectsProcessorDemo/Effects View Controllers/VariableDelayViewController.m
//
//  VariableDelayViewController.m
//  EffectsProcessorDemo
//
//  Created by Aurelius Prochazka on 1/29/15.
//  Copyright (c) 2015 Aurelius Prochazka. All rights reserved.
//

#import "VariableDelayViewController.h"
#import "SharedStore.h"

@implementation VariableDelayViewController {
    IBOutlet AKPropertySlider *delayTimeSlider;
    IBOutlet AKPropertySlider *mixSlider;
}

- (void)viewDidLoad
{
    SharedStore *global = [SharedStore globals];

    delayTimeSlider.property = global.variableDelay.delayTime;
    mixSlider.property       = global.variableDelay.mix;
}

@end

File: iOS Examples/Swift/EffectsProcessorDemo/EffectsProcessorDemo/VariableDelayViewController.swift
//
//  VariableDelayViewController.swift
//  EffectsProcessorDemo
//
//  Created by Matthew Fecher on 5/23/15.
//  Copyright (c) 2015 audiokit.io. All rights reserved.
//

import UIKit

class VariableDelayViewController: UIViewController {

    @IBOutlet weak var delayTimeSlider: AKPropertySlider!
    @IBOutlet weak var mixSlider: AKPropertySlider!
    
    override func viewDidLoad() {
        super.viewDidLoad()

        let global = SharedStore.globals
        delayTimeSlider.property = global.variableDelay!.delayTime
        mixSlider.property = global.variableDelay!.mix
    }



}
